README.md

```diff
@@ -1,3 +1,7 @@
+# Trello Link 
+
+https://trello.com/b/6FZkyg66/lambdamud-wonjae-hwang
```

Comment:
Good that you added your trello link here, it's a small thing but many students forget and it's important for documenting your code.